org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/impl/XtextBuilder.java

```diff
@@ -755,5 +761,38 @@ public ISchedulingRule getRule(int kind, Map<String, String> args) {
 			default: throw new IllegalArgumentException();
 		}
 	}
+	
+	/**
+	 * @since 2.27
+	 */
+	public void triggerRequestProjectRebuild() {
```

Comment:
It might make sense to accept an IResource or a URI as an argument to be able to request rebuilt for the project into which the file was written / from which the file was removed.

Reply:
the problem is that the postprocessor api is really bad for it. 
also generating to different projects does not work ootb. i would skip this for now